runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -655,8 +661,9 @@ private void addCounter()
 				+ " " + initialAmount;
 		}
 
+		int streak = getIntProfileConfig(SlayerConfig.SLAYER_MASTER_KEY) == 7 ? getIntProfileConfig(SlayerConfig.WILDERNESS_STREAK_KEY) : getIntProfileConfig(SlayerConfig.STREAK_KEY);
```

Comment:
I assume the `7` here means Krystilia, as commented in `Varbits.java`. Using an enum instead of a "magic number" would significantly improve the maintainability of this code.

Reply:
I thought it was fine before since it is explained in ``Varbits.java``,the plugin already contains [this](https://github.com/runelite/runelite/blame/664d479fdc646dee1adb3e7b910ff81adbf9af8d/runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java#L379), and it was only used once. Added it a static final int for now.